Module.php

```diff
@@ -144,8 +144,9 @@ public function bootstrap($app)
         $app->on(Application::EVENT_BEFORE_REQUEST, function () use ($app) {
             $app->getView()->on(View::EVENT_END_BODY, [$this, 'renderToolbar']);
         });
-
-        $app->getUrlManager()->addRules([
+        $urlManager = $app->getUrlManager();
+        $urlManager->ruleConfig['class'] = 'yii\web\UrlRule';
```

Comment:
Could you elaborate, please, why did you decided to change the `ruleConfig` class of the existing `UrlManager` component? I think it might break the `UrlManager` component in case when developer defined another `class` in the application config.